migrace/018.php

```diff
@@ -0,0 +1,10 @@
+<?php
+
+$this->q("
+
+UPDATE akce_seznam SET typ=13 where typ=14;
+DELETE FROM akce_typy where id_typu=14;
+UPDATE akce_typy SET typ_1pmn='Parcon a přednášky',typ_1p='Parcon nebo přednáška' WHERE id_typu=3;
```

Comment:
Nemění se všechny názvy => na některých místech (pravděpodobně v adminu někde kolem aktivit) přilepené info "a parcon" zmizí, zůstanou jenom přednášky. Pokud o tom Darien ví a bude to zadávat do přednášek, je to asi ok.